lib/evaluate.js

```diff
@@ -271,7 +275,7 @@ export const runFraudDetection = async ({
   //    or missing some of the required fields like `inet_group`
   //
   for (const m of measurements) {
-    if (m.fraudAssessment) continue
+    if (m.taskingEvaluation) continue
 
```

Comment:
I believe in this first step no measurement can yet have a `takingEvaluation`

Reply:
Good catch. This was introduced by https://github.com/filecoin-station/spark-evaluate/pull/439.

I'll remove this condition.

Reply:
See 1aba26794